sympy/stats/tests/test_continuous_rv.py

```diff
@@ -351,9 +351,10 @@ def test_gamma_inverse():
     b = Symbol("b", positive=True)
 
     X = GammaInverse("x", a, b)
+    Y = GammaInverse("y", 1, 1)
     assert density(X)(x) == x**(-a - 1)*b**a*exp(-b/x)/gamma(a)
     assert cdf(X)(x) == Piecewise((uppergamma(a, b/x)/gamma(a), x > 0), (0, True))
-
+    assert sample(Y) in Y.pspace.domain.set
```

Comment:
Make it a separate test (for Scipy-dependent sampling) and skip the test if import fails, as is done for example here:
https://github.com/sympy/sympy/blob/master/sympy/core/tests/test_numbers.py#L1814